flake.nix

```diff
@@ -86,5 +86,16 @@
                 srcRepo = lib.strings.hasInfix "snapshot" version;
               })
             versions);
+
+    apps = nixpkgs.lib.genAttrs [
```

Comment:
You can specify `meta.mainProgram` in each derivation, so you don't have to define apps: https://nixos.org/manual/nixpkgs/stable/#var-meta-mainProgram

Reply:
Ooh, nice